resources/definitions/lfextruder.def.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "LayerFused",
```

Comment:
There is no machine "LayerFused". This should refer to the file name of one of your printer definitions).